charts/neuvector/1.8.901+up1.8.9/templates/crd.yaml

```diff
@@ -0,0 +1,926 @@
+{{- if .Values.crdwebhook.enabled -}}
```

Comment:
The CRDs should be split into a secondary chart. It sounds like there is already a CRD-subchart separately, so that can be moved into the second chart. I'll link more info.

Reply:
I believe you can use `generateCRDChart` to do this still. Check other charts package.yaml.

Reply:
There is a separate CRD chart, the purpose of that chart is to allow users to install CRD before installing NeuVector. The core chart has everything, so they can do the installation in one shot.